src/liquid/components/ld-sidenav/ld-sidenav-back/ld-sidenav-back.css

```diff
@@ -0,0 +1,135 @@
+.ld-sidenav-back {
+  /* layout */
+  --ld-sidenav-back-border-radius: var(--ld-br-l);
+  --ld-sidenav-back-icon-size: var(--ld-sp-40);
+
+  /* colors */
+  --ld-sidenav-back-col: var(--ld-col-neutral-800);
+  --ld-sidenav-back-indicator-col: transparent;
+
+  outline: none;
+  display: block;
+
+  @media (hover: hover) {
+    &:where(:hover) {
+      --ld-sidenav-back-col: var(--ld-thm-primary);
+      --ld-sidenav-back-indicator-col: var(--ld-col-neutral-300);
```

Comment:
You are right. Though in a themed subnavigation (which we wanted to allow via a prop) it looks better with alpha (in my opinion). I'll leave this as is for now and see what @NicolasBlock91 says in the design review.

Reply:
Ok, we should explicitly point him at this, though, because he'd need to update the designs accordingly to make them match. (Which is probably necessary to prevent followup issues, as we're using color variables here.)

Reply:
Yeah, we should. I'll implement the prop in the next PR and give @NicolasBlock91 a heads-up.